src/components/shared/Pagination/Pagination.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Pagination } from './Pagination';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('Pagination Component', () => {
+  afterEach(cleanup);
+
+  it('should show previous page element when current page is not the first one', () => {
+    //Act
+    const { container } = render(
+      <BrowserRouter>
+        <Pagination currentPage={2} pagesCount={100} urlToGo={'https://domain.com/section?'} />
+      </BrowserRouter>,
+    );
+
+    // Asserts
+    expect(container.querySelector('.icon-arrow-prev')).toBeInTheDocument();
```

Comment:
I think here you do both validations for prev and next right?
```suggestion
    expect(container.querySelector('.icon-arrow-prev')).toBeInTheDocument();
    expect(container.querySelector('.icon-arrow-next')).toBeInTheDocument();
```

Reply:
Mmmmm, no. I have a validation for next element in another test.

Reply:
Yeah I know, what I'm saying is perhaps it can be done all in the same test right?

Reply:
Yes, but I preferred to keep them separated because the conditions to show each ... is different in each case.